lib/babe/verify_test.go

```diff
@@ -550,23 +552,20 @@ func Test_verifier_verifyAuthorshipRight(t *testing.T) {
 	babePrd2, err := testBabePrimaryPreDigest.ToPreRuntimeDigest()
 	assert.NoError(t, err)
 	header4 := newTestHeader(t, *babePrd2)
-
 	signAndAddSeal(t, kp, header4, []byte{1})
 	babeVerifier2 := newTestVerifier(kp, mockBlockState, scale.MaxUint128, false)
 
 	// Case 5: Invalid signature - BabeSecondaryPlainPreDigest
 	babeSecPlainPrd, err := testBabeSecondaryPlainPreDigest.ToPreRuntimeDigest()
 	assert.NoError(t, err)
 	header5 := newTestHeader(t, *babeSecPlainPrd)
-
 	signAndAddSeal(t, kp, header5, []byte{1})
 	babeVerifier3 := newTestVerifier(kp, mockBlockState, scale.MaxUint128, true)
 
 	// Case 6: Invalid signature - BabeSecondaryVrfPreDigest
 	encSecVrfDigest := newEncodedBabeDigest(t, testBabeSecondaryVRFPreDigest)
 	assert.NoError(t, err)
 	header6 := newTestHeader(t, *types.NewBABEPreRuntimeDigest(encSecVrfDigest))
-
```

Comment:
nit a lot of empty lines removed, not sure why. Try being careful next time to reduce deltas :wink:

Reply:
@timwu20 I modify the code based on what seems appropriate. Is that what I am supposed to do or write code based on how I create the least amount of deltas?